CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing to Monoweb
+
+We welcome contributionsto Monoweb. Please read [the developer documentation](docs/developer-guide.md) to get started.
```

Comment:
Maybe move the entire developer guide there yes, but I don't think we can put everything in the docs folder in this one file. It will be come unmanagebly huge in the future